public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -48,6 +48,25 @@ export interface DashboardLink {
   targetBlank: boolean;
 }
 
+type AnnotationType = 'tags' | 'dashboard';
+
+/**
+ * Typings for Annotation is added only for the basic scenarios.
+ * We need to improve the typings to support custom annotations
+ * added by different types of datasources.
+ */
+export interface Annotation {
```

Comment:
Instead of adding this use this type  https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/annotations.ts#L10